src/act_info.cpp

```diff
@@ -230,7 +230,9 @@ void show_char_to_char_0(const Char *victim, const Char *ch) {
         else if (victim->fighting == ch)
             buf += " |RYOU!|w";
         else if (victim->in_room == victim->fighting->in_room) {
-            buf += fmt::format(" {}.", pers(victim->fighting, ch));
+            std::string_view result;
+            result = ch->describe(*victim->fighting);
```

Comment:
nitpick: the assignment could be one line up.

Reply:
not a nitpick! that's the automatic tooling going wrong... there shouldn't be an intermediate variable at all.